docs/articles/documentation/recipes/debug-in-webstorm.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: docs
+title: Debug in WebStorm
+permalink: /documentation/recipes/debug-in-webstorm.html
+---
+# Debug in WebStorm
+
+This topic describes how to debug your TestCafe tests in [WebStorm](https://www.jetbrains.com/webstorm/) with a Node.js debugger.
+
+## Step 1 - Install TestCafe
+
+Execute the following command:
```

Comment:
Combine this sentence and the next one: 'Execute the following command to install TestCafe locally and add it to the `devDependencies` section in `package.json`:'